packages/langium/src/test/langium-test.ts

```diff
@@ -30,7 +30,27 @@ export function parseHelper<T extends AstNode = AstNode>(services: LangiumServic
     };
 }
 
-export type ExpectFunction = (actual: unknown, expected: unknown) => void;
+export type ExpectFunction = (actual: unknown, expected: unknown, message?: string) => void;
+
+let expectedFunction: ExpectFunction = (actual, expected, message) => {
+    if (typeof expect === 'function') {
+        if (message && expect.length === 2) {
+            // With `jest-expect-message`
+            expect(actual, message).toEqual(expected);
+        } else {
+            expect(actual).toEqual(expected);
+        }
+    } else {
+        throw new Error('No expect function provided');
+    }
+};
+
+/**
+ * Overrides the assertion function used by tests. Tries to use Jest by default.
+ */
+export function expectFunction(functions: ExpectFunction): void {
```

Comment:
The idea was that users of Langium can use any test framework they like, by calling this function with the respective assertion function. Since we use jest/vitest, there is no need for us to call this function, but there might be by downstream users.

Reply:
I see.